app/views/layouts/application.html.haml

```diff
@@ -10,14 +10,20 @@
     %meta{charset: "utf-8"}/
     %meta{content: "IE=edge", "http-equiv" => "X-UA-Compatible"}/
     %meta{content: "width=device-width, initial-scale=1", name: "viewport"}/
+    %meta{name: "frontend/config/environment", content: "%7B%22modulePrefix%22%3A%22frontend%22%2C%22environment%22%3A%22development%22%2C%22rootURL%22%3A%22%2F%22%2C%22locationType%22%3A%22none%22%2C%22EmberENV%22%3A%7B%22FEATURES%22%3A%7B%7D%2C%22EXTEND_PROTOTYPES%22%3A%7B%22Date%22%3Afalse%7D%2C%22_JQUERY_INTEGRATION%22%3Afalse%7D%2C%22APP%22%3A%7B%22name%22%3A%22frontend%22%2C%22version%22%3A%220.0.0%2Bc85e808a%22%7D%2C%22exportApplicationGlobal%22%3Atrue%2C%22isModuleUnification%22%3Afalse%7D"}/
```

Comment:
One downside I noted was that this meta tag includes the compiled config/environment.js file. Meaning as soon as you change the file, you will need to transfer the tag from the newly compiled index.html to the application.html.haml

Reply:
I see .... this seems to be generated by broccoli dev server.  since we are not going to change the environment.js every day, this is pretty ok.
maybe leave a note in environment.js ?